content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
 keywords: [kubernetes,vms]
 ---
 
-{{< warning_icon >}} Mesh expansion is broken in the initial release of Istio 1.0. We expect to have a patch available that
-fixes this problem very soon. We apologize for the inconvenience.
-
-Instructions for integrating VMs and bare metal hosts into an Istio mesh
+This guide provides instructions for integrating VMs and bare metal hosts into an Istio mesh
 deployed on Kubernetes.
 
 ## Prerequisites
 
-* Setup Istio on Kubernetes by following the instructions in the [Installation guide](/docs/setup/kubernetes/quick-start/).
+* You have already set up Istio on Kubernetes. If you haven't done so already, you can find out how in the [Installation guide](/docs/setup/kubernetes/quick-start/).
 
-* The machine must have IP connectivity to the endpoints in the mesh. This
+* Mesh expansion machines must have IP connectivity to the endpoints in the mesh. This
 typically requires a VPC or a VPN, as well as a container network that
 provides direct (without NAT or firewall deny) routing to the endpoints. The machine
 is not required to have access to the cluster IP addresses assigned by Kubernetes.
 
-* The Istio control plane services (Pilot, Mixer, Citadel) and Kubernetes DNS server must be accessible
-from the VMs. This is typically done using an [Internal Load
-Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).
-You can also use NodePort, run Istio components on VMs, or use custom network configurations,
-separate documents will cover these advanced configurations.
+* Mesh expansion VMs must have access to a DNS server that resolves names to cluster IP addresses. Options
+include exposing the Kubernetes DNS server through an internal load balancer, using a Core DNS
+server, or configuring the IPs in any other DNS server accessible from the VM.
+
+* If you haven't already enabled mesh expansion [at install time with Helm](/docs/setup/kubernetes/helm-install/), you have installed the [Helm client](https://docs.helm.sh/using_helm/). You'll need it to enable mesh expansion for the cluster.
```

Comment:
No, that's not what I meant - either you have enabled mesh expansion at install time (so you don't need to do the step involving the Helm client because it's already enabled), or you need to have the Helm client installed as a prerequisite. There aren't any missing conjunctions.